src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+
+interface Props {
+  errorMessage: string;
+  onClose: () => void;
+}
+
+export const Notification: React.FC<Props> = ({ errorMessage, onClose }) => {
+  const [showNotification, setShowNotification] = useState(false);
```

Comment:
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig
```suggestion
  const [isShowNotification, setIsShowNotification] = useState(false);
```